saleor/graphql/checkout/mutations/checkout_lines_add.py

```diff
@@ -185,6 +188,21 @@ def clean_input(
                 variants_ids_to_validate, checkout.channel_id
             )
 
+    @classmethod
+    def _validate_lines_metadata(cls, lines: list[CheckoutLineInput]):
+        try:
+            for line in lines:
+                metadata_manager.create_from_graphql_input(line.metadata)
+        except metadata_manager.MetadataEmptyKeyError:
+            raise ValidationError(
+                {
+                    "metadata": ValidationError(
+                        "Metadata key cannot be empty.",
+                        code=MetadataErrorCode.REQUIRED.value,
+                    )
+                }
+            ) from None
```

Comment:
```suggestion
            )
```

Reply:
why?
IIRC
- it's a good practice, to be explicit we don't leak the cause?
- We have ruff rule that is enforcing that